rebar.config

```diff
@@ -40,7 +36,7 @@
 
 {cover_opts, [verbose]}.
 
-{alias, [{test, [compile, format, lint, {ct, "--verbose"}, cover, edoc]}]}.
```

Comment:
This is a part of test for check if documentation is still generated as expected, the `lint` is also needed - please revert back this changes

Reply:
I see that `https://raw.githubusercontent.com` is a base URL, try move to macros URL the `https://raw.githubusercontent.com` maybe this will help

Reply:
This will help the linter, but they will become unclickable.